### PR TITLE
SUPNXP-14079: API for Register folder link for favourites has some in…

### PR DESCRIPTION
**Pull request by rsoumyassdi**
…consistencies

### CODE REVIEW

File: nuxeo-drive-client/nxdrive/osi/windows.py [hunk 1]
```diff
@@ -166,7 +166,7 @@ def unregister_contextual_menu(self):
             _winreg.DeleteKey(reg, self.get_menu_key())
             _winreg.DeleteKey(reg, self.get_menu_parent_key())
 
-    def register_folder_link(self, name, folder_path):
+    def register_folder_link(self, folder_path, name=None):
```
Comment (rsoumyassdi) [inline nuxeo-drive-client/nxdrive/osi/windows.py]: This method is still unused in nuxeo-drive

File: nuxeo-drive-client/nxdrive/osi/windows.py [hunk 2]
```diff
@@ -192,7 +192,7 @@ def _get_desktop_folder(self):
         from win32com.shell import shell, shellcon
         return shell.SHGetFolderPath(0, shellcon.CSIDL_DESKTOP, 0, 0)
 
-    def _create_shortcut(self, link, filepath, iconpath, description=None):
+    def _create_shortcut(self, link, filepath, iconpath=None, description=None):
```
Comment (rsoumyassdi) [inline nuxeo-drive-client/nxdrive/osi/windows.py]: This method is still unused in nuxeo-drive

**Comment by loopingz** [comment]
This api is first specified in the __init__.py and needs to be update there, in windows and darwin to be consistent

**Comment by rsoumyassdi** [comment]
Thanks remi for your review comments , I have updated the code with the changes suggested.

**Comment by loopingz** [comment]
You are missing the darwin.py modification

**Comment by rajadg** [comment]
The darwin.py modification is also done and code committed.

**Comment by loopingz** [comment]
Merged

**Pull request closed without merging**